src/components/Chat.js

```diff
@@ -86,9 +86,9 @@ class TextInput extends React.Component {
 
 
 const StyledTextComposer = styled.div`
-  padding:0.5em;
```

Comment:
Why remove padding?

Reply:
It is make hight diff between the send button and the text area. 
If it is require the better way would have a wrapper on both text area and send button and applied it there.
WDYS?